tests/capi/unittest_capi_service.cc

```diff
@@ -0,0 +1,78 @@
+/**
+ * @file        unittest_capi_service.cc
+ * @date        08 Mar 2022
+ * @brief       Unit test for ML Service C-API.
+ * @see         https://github.com/nnstreamer/api
+ * @author      Sangjung Woo <sangjung.woo@samsung.com>
+ * @bug         No known bugs
+ */
+
+#include <gtest/gtest.h>
+#include <glib.h>
+#include <ml-api-internal.h>
+#include <ml-api-service.h>
+
+/**
+ * @brief Test NNStreamer Service API for adding and getting the pipeline description with name
+ */
+TEST (nnstreamer_capi_service, basic_test_0_p)
```

Comment:
Please add more negative test cases.

Reply:
Thanks for your comments.
I will do that as a follow-up `Best Reviewer task`. 
- Add more positive and negative test cases
- Refactoring the code.